src/assignment/modals/DeleteAssignmentModal.tsx

```diff
@@ -42,6 +51,18 @@ const DeleteAssignmentModal: FunctionComponent<DeleteAssignmentModalProps> = ({
 			);
 		}
 
+		if (buildType) {
+			return tHtml(
+				'assignment/views/assignment-overview___deze-opdracht-bevat-mogelijk-collecties-die-eveneens-verwijderd-zullen-worden'
+			);
+		}
+
+		if (responses) {
+			return tHtml(
+				'assignment/views/assignment-overview___leerlingen-bekeken-deze-opdracht-reeds'
+			);
+		}
```

Comment:
what if the assignment is shared and has responses?
Then the user only sees the contributor warning message?

maybe you'll have to change this to:
```typescript
let contributorMessage = null;
let responsesMessages = null;

if (isSharedWithOthers) {
	if (contributorCount === 1) {
		contributorMessage = tHtml(
			'Ben je zeker dat je jezelf van deze opdracht wil wissen? Deze opdracht is met 1 andere persoon gedeeld. Deze verliezen dan toegang.'
		);
	} else {
		contributorMessage  = tHtml(
			'assignment/modals/delete-assignment-modal___ben-je-zeker-dat-je-jezelf-van-deze-opdracht-wil-wissen-deze-opdracht-is-met-count-andere-mensen-gedeeld-deze-verliezen-dan-toegang',
			{ count: contributorCount }
		);
	}
} else if (isContributor) {
	contributorMessage  = tHtml(
		'assignment/modals/delete-assignment-modal___ben-je-zeker-dat-je-jezelf-van-deze-opdracht-wil-wissen'
	);
}

if (responses) {
	if (buildType) {
		responsesMessages = tHtml(
			'assignment/views/assignment-overview___deze-opdracht-bevat-mogelijk-collecties-die-eveneens-verwijderd-zullen-worden'
		);
	} else {
		responsesMessages  = tHtml(
			'assignment/views/assignment-overview___leerlingen-bekeken-deze-opdracht-reeds'
		);
	}
}

return <>
	{contributorMessage}
	{responsesMessages}
</>
```